archetype-common/src/test/java/org/apache/maven/archetype/old/ArchetypeTest.java

```diff
@@ -363,4 +366,19 @@ protected void setUp()
         super.setUp();
         archetype = (OldArchetype) lookup( OldArchetype.ROLE );
     }
+
```

Comment:
I prefer using xmlunit over this code: xml comparison is tricky and xmlunits feedback on assert failures is quite good.

Reply:
@rfscholte 
Nice library, let me try that!